### DISCUSSION TITLE
Underline highlighted words

### DISCUSSION THREAD

**Discussion by LudwigCodes** [Chatterino/chatterino2#3819] [Ideas]
Hey,

i would like that you can add maybe that feature, if a word got highlighted, it got underlined as well. Chatty got this feature and i think it is usefull for bigger messages to understand the context without read the whole message. Or if not underline than make a frame around  

![chatterino](https://user-images.githubusercontent.com/77961307/173505872-7d067e03-0625-49d2-aadf-247081f89fe5.png)

**Comment by pajlada** [comment]
Somehow marking the highlighted word/user would be very nice, I'm not 100% sure Underlining would be the way we'd go but I'd happily accept PRs that go towards this feature.
